content/about.md

```diff
@@ -13,13 +13,32 @@ date = "2020-04-11"
 author = "Author1"
 +++ -->
 
-# Hi there
+# History
+**Null** or **n|u** is a registered not-for-profit society and an active security community with 13 chapters in major cities-
```

Comment:
"n|u" is the logo of the main organization, I don't think it should go in here

Reply:
this has been removed